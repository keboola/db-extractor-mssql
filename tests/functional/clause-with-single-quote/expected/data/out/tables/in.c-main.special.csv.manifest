{"destination":"in.c-main.special","incremental":false,"columns":["usergender","sku"]}