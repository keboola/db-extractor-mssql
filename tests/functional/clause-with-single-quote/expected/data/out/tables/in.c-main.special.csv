Male,ZD111402
Male,ZD111483
Male,ZD111451
Male,ZD111471
Male,ZD111228
